### PR TITLE
 Create Unit test suite #768

### PR DESCRIPTION
**Pull request by erangeles**

### CODE REVIEW

File: tests/unit/components/sl-radio-test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import{ moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-radio', 'Unit | Component | sl radio', {
+    unit: true
+});
+
+test( 'Correct default property values', function( assert ) {
```
Comment (notmessenger) [inline tests/unit/components/sl-radio-test.js]: Should also assert value of `tagName` property

File: tests/unit/components/sl-radio-test.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    );
+
+    assert.strictEqual(
+        component.get( 'value' ),
+        null,
+        'Default property "value" is null'
+    );
+});
+
+test( 'Inline property sets relevant class', function( assert ) {
```
Comment (notmessenger) [inline tests/unit/components/sl-radio-test.js]: * This test should be testing both conditions:  "radio-inline" when inline, "radio" default
* The `_dependentKeys` property should be used to confirm the correct properties are being observed.  A demo of this property's use (not in a test though approach is the same) can be viewed at http://jsbin.com/zituta/edit?html,js,output
* This computed property should be tested via calling it, such as via `this.get( 'radioType` )`.  Integration tests are about the rendering of the component and whether the template is configured as expected, unit tests are about the output of the function.

**Comment by erangeles** [comment]
@notmessenger having issues obtaining `_dependentKeys` as it's an "internal" variable. I only get the value of the property when I use the `this.get( 'radioType' )`. everything else has been addressed.

**Comment by notmessenger** [comment]
@erangeles

Does this work?

```
const component = this.subject();

component.radioType._dependentKeys
```

**Comment by erangeles** [comment]
@notmessenger all comments have been address; PR is ready to be reviewed again.

**Pull request merged**